plugin/output/kafka/kafka_test.go

```diff
@@ -0,0 +1,105 @@
+//go:build fuzz
+
+package kafka
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/Shopify/sarama"
+	"github.com/ozontech/file.d/pipeline"
+	insaneJSON "github.com/vitkovskii/insane-json"
+	"go.uber.org/zap/zaptest"
+)
+
+const (
+	defaultTopic = "default_topic"
+	topicField   = "pipeline_kafka_topic"
+)
+
+type mockProducer struct {
+	t *testing.T
+}
+
+func (m *mockProducer) ensureTopic(msg *sarama.ProducerMessage) {
+	val, err := msg.Value.Encode()
+	if err != nil {
+		m.t.Fatal(err)
```

Comment:
This verbose `err != nil` check can be replaced with `require.NoError(m.t, err)`.
require: https://pkg.go.dev/github.com/stretchr/testify/require#NoError

Reply:
Done